Frequently-Asked-Questions.mediawiki

```diff
@@ -129,7 +128,7 @@ Light theme:
 * Black for torrents in allocating or stalled(up and down) status
 * Royal Blue (rgb(65, 105, 225)) for torrents in uploading or forced uploading status
 * Salmon (rgb(250, 128, 114)) for torrents in paused(downloading) status
-* Dark Blue (rgb(0, 0, 139)) for torrents in completed (paused seeding) status
+* Dark Blue (rgb(0, 0, 139)) for torrents in completed(paused seeding) status
```

Comment:
I don't believe that omiting space in this case is correct punctuation.

Reply:
It's consistent with the other uses in this list (e.g. lines 131, 129) and the inconsistency stood out to me.  Should I change all the others to have a space as well?

Reply:
> Should I change all the others to have a space as well?

IMO, yes (except inappropriate cases like `rgb(0, 0, 139)`).

Reply:
Addressed in e4a6e9f